ui.apps/src/main/content/jcr_root/apps/venia/components/commerce/miniaccount/.content.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0"
     jcr:primaryType="cq:Component"
-    jcr:title="Mini Account"
+    jcr:title="Mini Cart"
```

Comment:
That shouldn't have changed, no?

Reply:
Yes I reverted it, this is wrong in the AEM archetype as well, created https://github.com/adobe/aem-project-archetype/pull/525 to fix it there as well.